src/test/java/org/minions/devfund/mauricioz/MovieDatabaseTest.java

```diff
@@ -0,0 +1,91 @@
+package org.minions.devfund.mauricioz;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * suit of unit tests for movie database class.
+ */
+public class MovieDatabaseTest {
+    private MovieDatabase movieDB;
+    private String movieName;
+    private String[] actor;
+    private Actor newActor;
+    private String mov1;
+    private String mov2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "mov2" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)